File: Modul-3/Soal-5/PRAK305-2310817110013-RANDY_FEBRIAN.py
for i in range(5):
    input_detik = int(input())
    hari, jam, menit, detik = 0, 0, 0, 0 
    while input_detik > 0:
        if input_detik >= 86400:
            hari = hari + 1
            input_detik = input_detik - 86400
        elif input_detik >= 3600:
            jam = jam + 1
            input_detik = input_detik - 3600
        elif input_detik >= 60:
            menit = menit + 1
            input_detik = input_detik - 60
        else:
            detik = detik + 1
            input_detik = input_detik - 1

    if hari >= 1:
        print("%d hari %.2d:%.2d:%.2d" % (hari, jam, menit, detik))
    else:
        print("%.2d:%.2d:%.2d" % (jam, menit, detik))

File: Modul-3/Soal-5/PRAK305-2310817110013-RANDY_FEBRIAN.c
#include <stdio.h>

int main(){
int input_detik;
    
    for(int i = 0; i < 5; i++){
        int hari = 0, menit = 0, detik = 0, jam = 0;
        scanf("%d", &input_detik);
        while(input_detik > 0){
            if(input_detik >= 86400){
                hari++;
                    input_detik = input_detik - 86400;
            }
            else if(input_detik >= 3600 && input_detik <= 86400.){
                jam++;
                input_detik = input_detik - 3600;

            }
            else if(input_detik >= 60 && input_detik <= 3600){
                menit++;
                input_detik = input_detik - 60;
            }
            else{
                detik++;
                input_detik--;
            }
        }

        if(hari >= 1){
            printf("%d hari %.2d:%.2d:%.2d\n", hari, jam, menit, detik);
        }
        else{
            printf("%.2d:%.2d:%.2d\n", jam, menit, detik);
        }
    }
    return 0;
}
